src/components/Block.jsx

```diff
@@ -20,20 +21,31 @@ module.exports = React.createClass({
     return app.refine('blockTypes').find(i => i.id === block.type)
   },
 
+  setMenuItems(component) {
+    if ('getMenuItems' in component) {
```

Comment:
naw, i just don't know much about when to check api settings like this vs. not. Would definitely throw a clear browser error.